models/classes/resources/Service/ClassCopier.php

```diff
@@ -96,9 +96,18 @@ public function copy(
     private function doCopy(
         core_kernel_classes_Class $class,
         core_kernel_classes_Class $destinationClass,
-        bool $keepOriginalPermission = true
+        bool $keepOriginalPermission = true,
+        string $copyIdentifier = '',
```

Comment:
I prefer to use `null` as default instead of empty string 🙂 
```suggestion
        ?string $copyIdentifier = null,
```